decoupled-site/react-script/src/graphql/OtherContentSearch.graphql

```diff
@@ -0,0 +1,26 @@
+query OtherContentSearch($searchParam: String, $locales: Locales!, $order: OrderBy) {
```

Comment:
Can we combine this with `ArtistSearch.graphql`? Perhaps call it `Search.graphql`?